winui/Avatar-view/content-types.md

```diff
@@ -0,0 +1,235 @@
+
+**DoubleCharacter**
+
+The double character initials type is used to display a two-character text as avatar that has been set in the `AvatarName` property. If the AvatarName consist of a single word, the first and last letters of that word will be displayed. However, if the `AvatarName` contains two or more words, the first letter of the first word and the first letter of the second word will be displayed.
+
+{% tabs %}
+{% highlight xaml %}
+
+<syncfusion:SfAvatarView ContentType="Initials"
+                         AvatarSize="ExtraLarge"
+                         AvatarName="Alex"
+                         InitialsType="DoubleCharacter"
+                         Background="Bisque">
+</syncfusion:SfAvatarView>
+
+{% endhighlight %}
+{% highlight c# %}
+          
+SfAvatarView avatarView = new SfAvatarView();
+avatarView.AvatarName = "Alex";
+avatarView.ContentType = AvatarContentType.Initials;
+avatarView.AvatarSize = AvatarSize.ExtraLarge;
+avatarView.InitialsType = AvatarInitialsType.DoubleCharacter;
+avatarView.Background = new SolidColorBrush(Colors.Bisque);
+
+{% endhighlight %}
+{% endtabs %}
+
+![WinUI AvatarView control with DoubleCharacter](avatarview_images/winui_double_character_initialstype_avatarview.png)
+
+## Custom image
+
+You can add any custom image as the avatar by using the `ImageSource` property.
+
+{% tabs %}
+{% highlight xaml %}
+
+<syncfusion:SfAvatarView ContentType="CustomImage"
+                         AvatarSize="ExtraLarge"
+                         ImageSource="Images\person.png">
+</syncfusion:SfAvatarView>
+
+{% endhighlight %} 
+{% highlight C# %}
+
+SfAvatarView avatarView = new SfAvatarView();
+avatarView.ContentType = AvatarContentType.CustomImage;
+avatarView.AvatarSize = AvatarSize.ExtraLarge;
+avatarView.ImageSource = new BitmapImage(new Uri("ms-appx:///Images\\person.png"));
+           
+{% endhighlight %}
+{% endtabs %}
+
+![WinUI AvatarView control with custom image](avatarview_images/winui_imagesource_avatarview.png)
+
+## Avatar character
+
+You can set the pre-defined avatar character images as avatar view by using the `AvatarCharacter` property.
+
+{% tabs %}
+{% highlight xaml %}
+
+<syncfusion:SfAvatarView ContentType="AvatarCharacter"
+                         AvatarSize="ExtraLarge"
+                         AvatarCharacter="Avatar15">
+</syncfusion:SfAvatarView>
+
+{% endhighlight %}
+{% highlight c# %}
+            
+SfAvatarView avatarView = new SfAvatarView();
+avatarView.ContentType = AvatarContentType.AvatarCharacter;
+avatarView.AvatarSize = AvatarSize.ExtraLarge;
+avatarView.AvatarCharacter = AvatarCharacter.Avatar15;
+            
+{% endhighlight %}
+{% endtabs %}
+
+![WinUI AvatarView control with avatar character](avatarview_images/winui_avatar_character_avatarview.png)
+
+## Group view 
+
+The `Group` content type allows you to display up to three images or initials within a single avatar view. To use this content type, you must set the following properties, which include:
```

Comment:
The `Group` content type allows you to display up to three images or initials within a single avatar view. To use this content type, you must set the following properties.